app/client/src/app/components/app-navbar/appNavbar.js

```diff
@@ -76,22 +76,18 @@ export default function AppNavbar() {
               Sponsor Us
             </Link>
           </Nav.Link>
-          {isLoggedIn ? (
-            <Nav.Link
-              className="hack-nav-item nav-non-button-item"
-              onClick={() => setExpanded(false)}
+          <Nav.Link
+            className="hack-nav-item nav-non-button-item"
+            onClick={() => setExpanded(false)}
+          >
+            <Link
+              to="/dashboard"
+              className={currentPath.includes("dashboard") && "selected"}
```

Comment:
Should this show when not logged in? Also, animation does not work for this button.